packages/components/src/disabled/test/index.js

```diff
@@ -133,6 +133,46 @@ describe( 'Disabled', () => {
 		expect( div.hasAttribute( 'tabindex' ) ).toBe( true );
 	} );
 
+	it( 'will disable descendant fields if isDisabled prop is set to true', () => {
```

Comment:
I think we can try to merge these two test cases together and simulate a state update from `isDisabled={true}` to `isDisabled={false}` in between. Just to make sure that the descendant fully updated.

Reply:
Isn't this already covered in the [previous test](https://github.com/WordPress/gutenberg/blob/add/components-disableable/packages/components/src/disabled/test/index.js#L99)?

Reply:
That test tested from wrapping `<Disabled>` to not wrapping it. What I expect here is to test it from `isDisabled={true}` to `isDisabled={false}` while always wrapping it with `<Disabled>`. Makes sense :)?

Reply:
Thanks for elaborating @kevin940726 - that makes sense! I'll update the tests shortly.

Reply:
Updated in 11c1a629a0f86f47b3c8afddedd6e4778a54290f. Thanks again!